cppcache/integration/test/RegisterKeysTest.cpp

```diff
@@ -112,6 +114,51 @@ TEST(RegisterKeysTest, RegisterAllWithCachingRegion) {
   }
 }
 
+TEST(RegisterKeysTest, RegisterAllWithConsistencyDisabled) {
+  Cluster cluster{LocatorCount{1}, ServerCount{1}};
+
+  cluster.start();
+
+  cluster.getGfsh()
+      .create()
+      .region()
+      .withName("region")
+      .withType("PARTITION")
+      .execute();
+
+  auto producer_cache = createTestCache();
+  auto listener_cache = createTestCache();
+
+  auto producer_region = [&cluster](Cache& cache) {
```

Comment:
Write it as a function. This looks like celebrating lambdas. A function that is only ever called in one place is inlined by all the major compilers. The lambda here obscures the code; you could have used a function with a useful name that tells me what you're doing, now I have an unnamed function body and I have to deduce it's meaning.